frontend/src/components/categories/index.tsx

```diff
@@ -166,24 +150,32 @@ export const Categories = () => {
                   startIcon={<AddIcon />}
                   variant="contained"
                   sx={{ float: "right" }}
-                  onClick={() => addDialogCtl.openDialog()}
+                  onClick={() => dialogs.open(CategoryFormDialog, null)}
                 >
                   {t("button.add")}
                 </Button>
               </Grid>
             ) : null}
-            {selectedCategories.length > 0 && (
-              <Grid item>
-                <Button
-                  startIcon={<DeleteIcon />}
-                  variant="contained"
-                  color="error"
-                  onClick={() => deleteDialogCtl.openDialog(undefined)}
-                >
-                  {t("button.delete")}
-                </Button>
-              </Grid>
-            )}
+            <Button
+              color="error"
+              variant="contained"
+              onClick={async () => {
+                const isConfirmed = await dialogs.confirm(
+                  <ConfirmDialogBody
+                    mode="selection"
+                    itemsNumber={selectedCategories.length}
+                  />,
+                );
```

Comment:
I would suggest to ensure consistency, just like we do `dialogs.open(Component)`, we should be consistent and do `dialogs.confirm(ConfirmDialogBody, {mode: 'selection', itemsNumber: selectedCategories.length}`

Reply:
I have updated the logic so we can pass extra options and follow the ensure consistency for dialogs